README.md

```diff
@@ -24,21 +24,38 @@ To run that demo on your own computer:
 * `npm run storybook`
 * Visit http://localhost:9001/
 
-## How to Use
+## Getting Started
+#### Install dependencies
+```
+npm install --save react-dates
+```
+Make sure you have the following dependencies installed:
+```
+ npm install --save moment react react-dom react-addons-shallow-compare
```

Comment:
looking into this further - it's possible yarn has better support for installing peer dependencies via the `--peer` flag, but I've also put together a Powershell script for the Windows users - should this be the expected output?

`npm install --save react-dates moment@"2.10-2.14||^2.15.1" react@">=0.14" react-addons-shallow-compare@">=0.14" react-dom@">=0.14"`

here is the script to produce that - it's not as clean and elegant as your unix script, but it gets the job done:

```powershell
$ErrorActionPreference = "SilentlyContinue"
$json = (npm info react-dates peerDependencies --json) | ConvertFrom-Json
$props = $json.data | Get-Member -membertype properties
$command = "npm install --save react-dates ";
foreach($prop in $props)
{
    $command += $prop.Name + "@`"" + (($json.data | Select -ExpandProperty $prop.Name) -replace '\s') + "`" "
}
Invoke-Expression $command
```